packages/create-bison-app/template/tests/jest.setup.js.ejs

```diff
@@ -6,13 +6,6 @@ const { nanoid } = require('nanoid');
 const exec = util.promisify(childProcess.exec);
 
 module.exports = async () => {
-  // console.log(`\n[jest] waiting for server's health check...`);
-  // let isUp = false;
-  // while (isUp === false) {
-  //   await new Promise((r) => setTimeout(r, 250));
-  //   isUp = await statusCheck();
-  // }
-  // console.log(`[jest] server is up...`);
```

Comment:
Do we ever use this? Are there better ways to wait for the server to start? Also, `statusCheck` isn't defined.